Applications/Hermes/hermesflaskexample_rx.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import base64
+import cPickle
+import time
+import ConfigParser
+import os
+
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("hermes.ini")
```

Comment:
Same reponse: Push to another update to add logging.

https://github.com/FaradayRF/Faraday-Software/pull/133#discussion_r108609659